File: Question-One/maintenance.js
let machines= [{name:"MachineA",personeel:2,historyRecord:4, demand:"High", parts:"Engine"},{name:"MachineB",personeel:4,historyRecord:4,demand:"low" , parts:"Lever"} ]

function checkPriority(){
    for (i=0;i<machines.length;i++ ){
        let machine =machines[i]

        if(machine.personeel<=4 && machine.historyRecord<=4 && machine.demand=="High" && machine.parts=="Engine"){
            console.log(`This ${machine.name} gets the priority  to be serviced`)}

        
      else if
      (machine.personeel>4 &&
       machine.personeel<10 &&
       machine.parts=="Lever"&&
        machine.historyRecord>4 && 
        machine.historyRecord<10){
        console.log(`This ${machine.name} gets mid priority  to be serviced`)

      }

      else{
        console.log(`This ${machine.name} gets the leaset priority  to be serviced`)
      }  
        
    }}
    checkPriority()

File: Question-One/maintenance.py
def maintainance_priority(machines):

   for machine in machines:
    
    if machine['personnel'] <= 4 and machine['historicaldata'] <= 4 and machine['demand'] == 'High' and machine['parts'] =='Engine':
        print(f'{machine["name"]} is of high priority')
    elif machine['personnel'] <= 3 and machine['historicaldata'] <= 5 and machine['demand'] == 'Low' and machine['parts']=='Brakes':
        print(f'{machine["name"]} is of low priority')
    else:
        print(f'{machine["name"]} doesn’t need service')
        
machines = [
    {"name": "MachineA", "personnel": 2, "historicaldata": 4, "demand": "High","parts":"Engine"},
    {"name": "MachineB", "personnel": 4, "historicaldata": 4, "demand": "Low","parts":"Brakes"}
]

maintainance_priority(machines)